project-koku/koku
# switching intial ingest to priority queue

Fixes [COST-2071](https://issues.redhat.com/browse/COST-2071)

Changes proposed in this PR:
* Switching to priority queue for initial ingest/summary to help new on boarding customers get there initial data sooner. 
* Really adding this for our QE tests so those don't randomly fail during a month when queue lengths are extremely long

Testing Instructions:
1. Add new source
2. See data process via priority queue
3. Also ensure day to day flow continues to not use priority queues.

NOTE:
Easiest way to test this is just stick `LOG.info(f"\n\n QUEUES: {SUMMARY_QUEUE}, {REPORT_QUEUE} \n\n")` after line 142, then just add a source see it log priority, then hit download again and see those change back to the usual queue names.
